src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+header {
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+nav {
+  margin: 0;
+}
+
+.nav-list {
+  height: 60px;
+  display: flex;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__item {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
```

Comment:
Be consistent with your *vertical* margins. Add only top or only bottom margin, but not both. Here you've added a `margin-right`. Consider if this is the best approach.